v2/tools/generator/internal/codegen/pipeline/determine_resource_ownership.go

```diff
@@ -58,75 +100,50 @@ func determineOwnership(
 		updatedDefs[resolved.SpecDef.Name()] = newDef
 	}
 
-	setDefaultOwner(configuration, definitions, updatedDefs)
+	if len(errs) > 0 {
```

Comment:
extremely minor:
Could also be:
```
err := kerrors.NewAggregate(errs)
if err != nil {
     errors.Wrapf(
			err,
			"failed to update ownership for some resources")
}
```

which I think is the pattern we usually use.

Reply:
Using `len(errs) > 0` is the usual way I've done it - lots of matches to `git grep "len(errs)"`